Procfile

```diff
@@ -0,0 +1 @@
+worker: node dist/index.js
```

Comment:
Yeah, I was thinking about that. I'll see if it is something that we can do as part of Heroku startup, but it might also be something we could put in a GitHub action or CI/CD pipeline if we get that far.